client/src/logic/local-settings/local-settings-reducer.ts

```diff
@@ -1,38 +1,58 @@
 import {LocalMessage, localMessages} from 'logic/messages'
 
-type LocalSettings = Record<string, any>
+export type LocalSettings = {
+	soundVolume: number
+	musicVolume: number
+	muted: boolean
+	profanityFilter: boolean
+	showChat: boolean
+	disableChat: boolean
+	confirmationDialogs: boolean
+	showBattleLogs: boolean
+	showAdvancedTooltips: boolean
+	chatPosition: {x: number; y: number}
+	chatSize: {w: number; h: number}
+	panoramaEnabled: boolean
+	panorama: string
+	gameSide: string
+	minecraftName: string
+}
+
+const defaultSettings: LocalSettings = {
+	soundVolume: 100,
+	musicVolume: 75,
+	muted: false,
+	profanityFilter: true,
+	disableChat: false,
+	confirmationDialogs: true,
+	showChat: false,
+	showBattleLogs: false,
+	showAdvancedTooltips: true,
+	chatPosition: {x: 0, y: 0},
+	chatSize: {w: 0, h: 0},
+	panoramaEnabled: true,
+	panorama: 'hermit-hill',
+	gameSide: 'Left',
+	minecraftName: 'alex',
```

Comment:
i would change some of these to be like `profanityFilterEnabled`

Reply:
Planning on doing this in a separate update, feel it's out of scope